packages/new-polymath-issuer/src/pages/Home.tsx

```diff
@@ -90,12 +92,24 @@ export class ContainerBase extends Component<Props> {
         name: 'My Dividend Distribution',
         amount: new BigNumber('10000'),
         checkpointId: 1,
-        excludedAddresses: [],
+        excludedAddresses: ['0x821aea9a577a9b44299b9c15c88cf3087f3b5544'],
+        // excludedAddresses: [],
```

Comment:
👁

Reply:
Remember that this is the home screen we use to test transactions. It's not really production code so I don't see the need to keep it clean

Reply:
Oh right, missed that. Nvm then!